tests/python/relay/test_tensorrt.py

```diff
@@ -0,0 +1,550 @@
+        def check_trt_used(graph):
+            import json
+            graph = json.loads(graph)
+            num_trt_subgraphs = sum([1 for n in graph['nodes'] if n.get('attrs', {}).get('func_name', '') == 'tensorrt_0'])
+            assert num_trt_subgraphs == 1
+
+        block = get_model(model, pretrained=True)
+        mod, params = relay.frontend.from_mxnet(block, shape={'data': input_shape}, dtype=dtype)
+
+        if use_trt:
+            mod = relay.tensorrt.EnableTrt(mod, params)
+            assert mod['main'].attrs and mod['main'].attrs.Compiler == 'tensorrt'
+            with relay.build_config(opt_level=2, disabled_pass={"SimplifyInference"}):
+                graph, lib, params = relay.build(mod, "cuda")
+            check_trt_used(graph)
+        else:
+            with relay.build_config(opt_level=3):
+                graph, lib, params = relay.build(mod, "cuda", params=params)
+
+        mod = graph_runtime.create(graph, lib, ctx=tvm.gpu(0))
+        mod.set_input(**params)
+        # Warmup
+        for i in range(10):
+            mod.run(data=i_data)
+
+        # Time
+        times = []
+        for i in range(num_iteration):
+            start_time = time.time()
+            mod.run(data=i_data)
+            res = mod.get_output(0)
+            times.append(time.time() - start_time)
+        latency = 1000.0 * np.mean(times)
+        print(model, latency)
+        return latency, res
+
+    latency = {}
+    models = [
+        'alexnet',
+        'resnet18_v1',
+        'resnet18_v2',
+        'squeezenet1.0',
+        'mobilenet0.25',
+        'mobilenetv2_0.25',
+        'vgg11',
+        'densenet121']
+    additional_models = [
+        'resnet34_v1',
+        'resnet50_v1',
+        'resnet101_v1',
+        'resnet152_v1',
+        'resnet34_v2',
+        'resnet50_v2',
+        'resnet101_v2',
+        'resnet152_v2',
+        'mobilenet0.5',
+        'mobilenet0.75',
+        'mobilenet1.0',
+        'mobilenetv2_0.5',
+        'mobilenetv2_0.75',
+        'mobilenetv2_1.0',
+        'vgg16',
+        'densenet169',
+        'densenet201']
+    if test_all_models:
+        models.extend(additional_models)
+    
+    dtype = 'float32'
+    input_shape = (1, 3, 224, 224)
+    i_data = np.random.uniform(0, 1, input_shape).astype(dtype)
+    for model in models:
+        latency[model], res = test_model(model, i_data, input_shape, dtype, use_trt=True)
+        # _, ref_res = test_model(model, i_data, input_shape, dtype, use_trt=False, num_iteration=1)
+        # tvm.testing.assert_allclose(res.asnumpy(), ref_res.asnumpy(), rtol=1e-3, atol=1e-3)
+    
+    for model in models:
+        print(model, latency[model])
+
+def test_tensorrt_serialize():
+    if not tvm.module.enabled("cuda") or not tvm.gpu(0).exist:
+        print("skip because cuda is not enabled.")
+        return
+    if not relay.tensorrt.IsTrtRuntimeAvailable():
+        print("skip because tensorrt runtime is not available")
+        return
```

Comment:
Two checks above repeat 5 times in this module. Should we have utility method for this?